allennlp/semparse/contexts/table_question_knowledge_graph.py

```diff
@@ -88,6 +164,10 @@ def __init__(self,
                  question_tokens: List[Token]) -> None:
         super().__init__(entities, neighbors, entity_text)
         self.question_tokens = question_tokens
+        entity_extractor = QuestionEntityExtractor( '/u/murtyjay/allennlp_fork/stop_words.json',  entity_text)
```

Comment:
How big is the stop words list? If it is not huge, may be you can define the list as a constant in the module instead of reading from a local file?

Reply:
there's 158 words in there. not too large, maybe I could define it as a constant somewhere